src/crepe/api/AI.cpp

```diff
@@ -0,0 +1,87 @@
+#include <stdexcept>
+
+#include "AI.h"
+
+namespace crepe {
+
+AI::AI(game_object_id_t id, float max_force) : Component(id), max_force(max_force) {}
+
+void AI::make_circle_path(float radius, const vec2 & center, float start_angle,
+						  bool clockwise) {
+	if (radius <= 0) {
+		throw std::runtime_error("Radius must be greater than 0");
+	}
+
+	// The step size is determined by the radius (step size is in radians)
+	float step = 400.0f / radius;
```

Comment:
wat is de 400?

Reply:
Een random waarde die testen het beste werkte... Dat is een nadeel van AI...

Reply:
Kun je dan er wel een const waarde van maken?